src/main/java/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobProperty.java

```diff
@@ -71,7 +71,7 @@ public PipelineTriggersJobProperty(List<Trigger> triggers) {
     }
 
     public void setTriggers(List<Trigger<?>> triggers) {
-        this.triggers = triggers;
+        this.triggers.addAll(triggers);
```

Comment:
Works out the same, I think - we init `this.triggers` anyway. But I can change it here if you'd like.

The `unmodifiableList` is off in [`PersistedList#toList`](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/util/PersistedList.java#L190)'s call to ['CopyOnWriteList#getView`](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/util/CopyOnWriteList.java#L145). I tend to think that probably should stay as is regardless.